health/notifications/telegram/README.md

```diff
@@ -9,13 +9,31 @@ Netdata will send warning messages without vibration.
 You need:
 
 1.  A bot token. To get one, contact the [@BotFather](https://t.me/BotFather) bot and send the command `/newbot`. Follow the instructions.
-2.  A chat id for every chat you want to send messages to. Contact the [@myidbot](https://t.me/myidbot) bot and send the command `/getid` to get your personal chat id or invite him into a group and issue the same command to get the group chat id.
-3.  Start a conversation with your bot or invite him into a group you want to sent messages to.
+2.  A chat id for every chat you want to send messages to. Contact the [@myidbot](https://t.me/myidbot) bot and send the command `/getid` to get your personal chat id or invite it into a group and issue the same command to get the group chat id. Group IDs start with a hyphen, supergroup IDs start with `-100`.
+3.  Start a conversation with your bot or invite it into a group you want to sent messages to.
 
-See slack for configuration.
+```
+SEND_TELEGRAM="YES"
+TELEGRAM_BOT_TOKEN="111122223:7OpFlFFRzRBbrUUmIjj5HF9Ox2pYJZy5"
+DEFAULT_RECIPIENT_TELEGRAM="-100233335555"
+```
+
+You can define multiple recipients like this: `"-100311112222 212341234|critical"`. 
+This example will send:
+
+-   All alerts to the group with ID -100311112222
+-   Critical alerts to the user with ID 212341234
+
+You can give different recipients per **role** using these (in the same file):
+
+```
+role_recipients_telegram[sysadmin]="212341234"
+role_recipients_telegram[dba]="-1004444333321"
+role_recipients_telegram[webmaster]="49999333322 -1009999222255"
+```
 
 Telegram messages look like this:
 
-![image](https://fb.hash.works/ytl/preview.jpg)
+![image](https://i.imgur.com/5AvoSI5.png)
```

Comment:
We prefer our images be hosted by GitHub's CDN. Here's the same image over there (plus an alt 😉):

```
![Netdata notifications via Telegram](https://user-images.githubusercontent.com/1153921/66612223-f07dfb80-eb75-11e9-976f-5734ffd93ecd.png)
```

Reply:
Thanks!